### PR TITLE
Add support for pinging IRC

### PR DESCRIPTION
**Pull request by Half-Shot**
This adds support for specifying an IRC channel to send pings and pongs to.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,16 +19,17 @@
   },
   "homepage": "https://github.com/t2bot/matrix-service-lag-bot#readme",
   "devDependencies": {
-    "@types/node": "^8.10.39",
-    "tslint": "^5.12.1",
-    "typescript": "^3.2.4"
+    "@types/node": "^14.0.0",
+    "tslint": "^6.1.3",
+    "typescript": "^4.3.3"
```
Comment (Half-Shot) [inline package.json:24]: I had to update the major version of this to be compatible with `matrix-org-irc`. TS looked happy afterwards.

File: src/discord/DiscordService.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ export class DiscordService implements IService {
         return chan.send(content);
     }
 
-    public waitForMessage(targetReference: any, content: string): Promise<any> {
+    public waitForMessage(targetReference: any, content: string): Promise<void> {
```
Comment (Half-Shot) [inline src/discord/DiscordService.ts:36]: These had to be changed since TS promises are now checked

**Review by Half-Shot — COMMENTED** [review]
(GitHub thinks I'm trying to review, oh well)

**Comment by Half-Shot** [comment]
@turt2live this is ready now.

**Review by turt2live — APPROVED** [review]

**Pull request merged**